functions/toCamelCase/toCamelCase.js

```diff
@@ -0,0 +1,13 @@
+export const toCamelCase = (str) => {
+  if (typeof str !== 'string') {
+    return '';
+  }
+
+  const words = str.split(/[-_]/);
+  const camelCaseText = words.map((word, index) => {
+    if (index === 0) return word;
+    return word.charAt(0).toUpperCase() + word.slice(1);
```

Comment:
Use a pattern string and interpolation instead of concatenation:
return `${someCode}${someCode}`;